src/ex3_js-objects-part1/task-03.js

```diff
@@ -0,0 +1,11 @@
+function isPropertyExists(stringArg, objectArg){
+    let result;
```

Comment:
`
    if (stringArg in objectArg){
        return true;
    }
    return false;
`